doc/rados/operations/cache-tiering.rst

```diff
@@ -549,6 +549,63 @@ disable and remove it.
 
       ceph osd tier remove cold-storage hot-storage
 
+Troubleshooting Unfound Objects
+===============================
+Under certain circumstances, restarting OSDs may result in unfound objects.
```

Comment:
If this is limited to cache tiering, we should include that condition here.  But haven't cache tiers been deprecated for several releases?